test_libs/pyspec/tests/test_sanity.py

```diff
@@ -438,7 +436,7 @@ def test_balance_driven_status_transitions(state):
     assert pre_state.validator_registry[validator_index].exit_epoch == spec.FAR_FUTURE_EPOCH
 
     # set validator balance to below ejection threshold
-    set_balance(pre_state, validator_index, spec.EJECTION_BALANCE - 1)
+    pre_state.validator_registry[validator_index].effective_balance = spec.EJECTION_BALANCE - 1
```

Comment:
Make this exactly `EJECTION_BALANCE` now that the equality condition is changed

```suggestion
    pre_state.validator_registry[validator_index].effective_balance = spec.EJECTION_BALANCE
```